eng/copy-wpf.ps1

```diff
@@ -0,0 +1,126 @@
+# Copy wpf binaries from local build to desired location. The location can either be the version number (for copying to shared installation)
+# or the location of the project file if copying binaries to the output of a locally built application.
+Param(
+[string]$destination,
+[string]$arch="x86",
+[switch]$release,
+[switch]$shared,
+[switch]$help
+)
+
+function CopyByBinaryToPublishLocation($publishLocation)
+{
+    $locallyBuiltBinaryLocationBase = Join-Path $RepoRoot "artifacts\bin"
+    CopyNativeBinariesToPublishLocation $publishLocation $locallyBuiltBinaryLocationBase 
+    CopyManagedBinariesToPublishLocation $publishLocation  $locallyBuiltBinaryLocationBase
+    CopyThemeBinariesToPublishLocation $publishLocation  $locallyBuiltBinaryLocationBase
+    CopyUIAutomationBinariesToPublishLocation $publishLocation  $locallyBuiltBinaryLocationBase
+}
+
+function CopyNativeBinariesToPublishLocation($publishLocation, $localBinLocation)
+{
+    $NativeBinaries = "D3DCompiler", "PenImc", "PresentationNative", "wpfgfx"
```

Comment:
All the relevant binaries are usually gathered under `artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub\lib\netcoreapp3.0` for easy packaging. You can probably gather them from there instead of having to maintain a hardcoded list here.

Reply:
Yeah that's what i had originally done, but i noticed that build -pack doesn't always update them. and it was one of those things that at the time i didn't want to go down the rabbit hole of figuring out was going on. i totally could be doing something wrong if no one else has seen this?

Reply:
I'm going to leave as is for now, I'd prefer to not have to rmdir, i do have a local version of this script that uses that directory that we can update once build -pack works as expected

Reply:
now that you have fixed the packaging issue i've gone and updated this :) thanks!